### PR TITLE
feat: add cd for pinner service

### PR DESCRIPTION
**Pull request by rafaiovlabs**
Adds CD workflow for pinner(s)

### CODE REVIEW

**Review by nduchak — APPROVED** [review]
LGTM

File: .github/workflows/staging.yml [hunk 1]
```diff
@@ -0,0 +1,70 @@
+---
+name: RIF Marketplace PINNER deploy
+
+on:
+  push:
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+-**'
+      - 'v[0-9]+.[0-9]+.[0-9]+'
```
Comment (0xartem) [inline .github/workflows/staging.yml:8]: The same thing here as in the UI, I don't think we should deploy tags to the staging env

File: .github/workflows/staging.yml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    branches:
+      - staging
+      - features/devops
+
+jobs:
+  deploy-staging:
```
Comment (0xartem) [inline .github/workflows/staging.yml:14]: Also, as in the UI PR, will we have jobs for the testnet and mainnet?

Comment (0xartem) [reply .github/workflows/staging.yml:14]: And what is the process of adding job for the develop branch? We can add it only after we have it set up in the infra?

**Review by 0xartem — CHANGES_REQUESTED** [review]

File: .github/workflows/staging.yml (hunk 2 above)
Comment (rafaiovlabs) [reply .github/workflows/staging.yml:14]: "Also, as in the UI PR, will we have jobs for the testnet and mainnet?" : We will have workflows for every environment, I'm working doing only staging CD to test and validate workflows that can later be propagated to the other Envs easily 
"And what is the process of adding job for the develop branch? We can add it only after we have it set up in the infra?" Yes each branch corresponds to an infra environment, so develop will be deployed on develop infrastructure, staging on staging infrastructure and so on, we still don't have dev env we cannot deploy it yet.

File: .github/workflows/staging.yml (hunk 1 above)
Comment (rafaiovlabs) [reply .github/workflows/staging.yml:8]: As we discussed for UI ..can we also merge this one ? .

**Review by 0xartem — APPROVED** [review]

**Comment by 0xartem** [comment]
@rafaiovlabs merged the development branch

**Pull request merged**